src/Components/CreateImageWizard/steps/googleCloud.js

```diff
@@ -111,7 +111,7 @@ const googleCloudStep = {
       component: componentTypes.RADIO,
       label: 'Select image sharing',
       isRequired: true,
-      name: 'image-sharing',
```

Comment:
The `image_sharing` on line 131 in CreateImageWizard.js could be changed to `image-sharing` so it's consistent with the other values using dashes and it wouldn't need to be changed in all the other places. 

Could you also please add a short commit message?